script/cargo.py

```diff
@@ -48,6 +48,7 @@ def build(args):
     # Set environment variables for rustup
     env = os.environ.copy()
 
+    rustup_root = rustup_home
```

Comment:
I assume this is using the pre-configured rustup installation downloaded, and not rustup installed locally?

Reply:
This is what have been passed into the script in accordance to 
`/brave/build/cargo.gni` configuration. Yes, this is where downloaded package exists.

Reply:
thanks